wordle-web/src/views/LeaderboardView.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <h1>Leaderboard</h1>
+  <v-row>
+    <v-col> Player Name: </v-col>
+    <v-col> Average Number of Attempts: </v-col>
+    <v-col> Average Seconds Per Game: </v-col>
+  </v-row>
+  <v-row v-for="player in leaderboard" :key="player['playerId']">
+    <v-col>
+      {{ player['name'] }}
+    </v-col>
+    <v-col>
+      {{ player['averageAttempts'] }}
+    </v-col>
+    <v-col>
+      {{ player['averageSecondsPerGame'] }}
+    </v-col>
+  </v-row>
```

Comment:
A different way to do this would be to create an interface to represent the data that is coming in so that way you get intellisense and you can use player.name, player. averageAttempts, etc.

Reply:
I'll check that out, thank you. (anything would probably be better than the way I was doing that)